cmd/run.go

```diff
@@ -459,8 +459,9 @@ func RunServer(ctx context.Context, config *Config) error {
 		}()
 	}
 
+	cachedOpenFGADatastore := caching.NewCachedOpenFGADatastore(datastore, config.Datastore.MaxCacheSize)
```

Comment:
Can you move this to where we initialize the datastore? Line 390 or so. Just so we can keep all the datastore initialization stuff close together.